runtime/make/Makefile.runtime.comm-ofi

```diff
@@ -15,10 +15,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ifneq ($(LIBFABRIC_DIR),)
-RUNTIME_INCLS += -I$(LIBFABRIC_DIR)/include
+ifeq ($(CHPL_MAKE_LIBFABRIC),system)
+  RUNTIME_INCLS += $(shell which pkg-config >& /dev/null && \
+                           pkg-config --exists libfabric && \
+                           pkg-config --cflags libfabric)
```

Comment:
Can this logic be pushed into third_party_utils.py and called from here? 

This looks really similar to `pkgconfig_get_link_args` and I'm wondering if we should have a `pkgconfig_get_cflags` or something

Reply:
Sure, I'll do that.

Reply:
The way we use `pkgconfig_get_link_args` is that we build up the link args in `printchplenv.py` and then print them as `CHPL_THIRD_PARTY_LINK_ARGS`, which `$CHPL_HOME/make/Makefile.base` turns into `CHPL_MAKE_THIRD_PARTY_LINK_ARGS`, and we add that whole thing to the linking command line.  I'm envisioning doing this the same way except using, say, `CHPL_THIRD_PARTY_COMPILE_ARGS`.  The alternative would be just a spot solution of invoking `pkgconfig_get_cflags` for the libfabric args here, but I prefer the former, more general solution.  (Though I don't think I'd tackle adding all the other applicable 3rd-party packages to it at this time.)  How about you?

Reply:
Done, in 9b1c868.